### DISCUSSION TITLE
About Laptops with Optimus and Stable's Compatibility Mode

### DISCUSSION THREAD

**Discussion by auroraongithub** [ppy/osu#21856] [Q&A] [answered]
I tried lazer for about thirty minutes, it was good but it could be better if there was no input lag. It was definitely running the same, maybe even more FPS than stable so I can see the performance updates were real and noticable at least lol. My laptop has a GTX 860M and an I7-4700HQ, and obviously, optimus is probably on but I can't be too bothered to turn it off (not very tech knowledgable person cosplay), I've already spent two hours just to get the game running on exclusive fullscreen to no avail. So the question is, when will I get to run lazer on exclusive fullscreen to not have as much input lag or get the same effect as Stable's Compatibility Mode?

**Comment by Theighlin** [comment, marked as answer]
> when will I get to run lazer on exclusive fullscreen to not have as much input lag or get the same effect as Stable's Compatibility Mode?

When the game has moved away from OpenGL, you can keep track of it over at https://github.com/ppy/osu-framework/issues/4909.

**Reply by auroraongithub** [reply]
Thanks (～￣▽￣)～
